doc/source/whatsnew/v0.23.0.txt

```diff
@@ -1080,6 +1080,7 @@ Performance Improvements
 - Improved performance of :func:`Series.isin` in the case of categorical dtypes (:issue:`20003`)
 - Improved performance of ``getattr(Series, attr)`` when the Series has certain index types. This manifiested in slow printing of large Series with a ``DatetimeIndex`` (:issue:`19764`)
 - Fixed a performance regression for :func:`GroupBy.nth` and :func:`GroupBy.last` with some object columns (:issue:`19283`)
+- Improved performance of :meth:`Categorical.loc` (:issue:`20395`)
```

Comment:
this doesn't exist, rather its indexing on a Series/DataFrame with a CategoricalIndex

Reply:
move to 0.23.1